test/unit/math/prim/prob/von_mises_cdf_test.cpp

```diff
@@ -0,0 +1,189 @@
+#include <stan/math/prim.hpp>
+#include <stan/math/prim/fun/constants.hpp>
+#include <gtest/gtest.h>
+
+TEST(ProbVonMises, random_cdf_test) {
+  using stan::math::von_mises_cdf;
+  double pi = stan::math::pi();
+  // check that our von_mises_cdf coincides with scipy's
+  double ABS_TOL = 1e-12;
+  ASSERT_NEAR(0.0000157567385742, von_mises_cdf(-3.141492653589793, 0, 0.01),
```

Comment:
The prim tests should test all the throws happen in the right cases: https://github.com/stan-dev/math/blob/f24720ca194057e7e27e8306cc6cb09a23c25b2f/stan/math/prim/prob/von_mises_cdf.hpp#L103

Like: https://github.com/stan-dev/math/blob/develop/test/unit/math/prim/fun/diag_post_multiply_test.cpp#L40